### PR TITLE
Added Dependencies To Issue Template And Merged Two Description Fields

### PR DESCRIPTION
**Pull request by garethellis0**
### Description

See PR title.

### Testing Done
None

### Resolved Issues

None

### Review Checklist

**_It is the reviewers responsibility to also make sure every item here has been covered_**

- [x] **Start of document comments**: each `.cpp` and `.h` file should have a comment at the start of it. See files in the `thunderbots/software/geom` folder for examples.
- [x] **Function comments**: All function definitions (usually in the `.h` file) should have a javadoc style comment at the start of them. For examples, see the classes defined in `thunderbots/software/geom`
- [x] **Remove all commented out code**
- [x] **Remove extra print statements**: for example, those just used for testing
- [x] **Resolve all TODO's**: All `TODO` (or similar) statements should either be completed or associated with a github issue
- [x] **Justify drops in code coverage**: If this PR results in a non-trivial drop in code coverage (a bot should post a coverage diagram as a comment), please justify why we can't test the code that's not covered.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/ticket.md [hunk 1]
```diff
@@ -50,3 +30,12 @@ about: Defines an item of work for the project
 - [ ] Item A
 - [ ] Item B
```
Comment (jonathanlew) [inline .github/ISSUE_TEMPLATE/ticket.md:31]: Delete Item B and C. We just need Item A as template

**Review by jonathanlew — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/ticket.md (hunk 1 above)
Comment (garethellis0) [reply .github/ISSUE_TEMPLATE/ticket.md:31]: I like having a few, as it more clearly indicates it's a series. Definitely a subjective thing though, so perhaps other people can weigh in here?

File: .github/ISSUE_TEMPLATE/ticket.md [hunk 2]
```diff
@@ -50,3 +30,12 @@ about: Defines an item of work for the project
...
 - [ ] Item C
+
+### Dependencies
+
+<!--
+    List all other issues this depends on, ex:
+    - #123
```
[the remark this answers is not in the document]
Comment (garethellis0) [reply .github/ISSUE_TEMPLATE/ticket.md:38]: Agreed, changed section to be named `Blocked By` and revised comment wording.

**Review by akhilveeraghanta — APPROVED** [review]
LGTM 🎉

File: .github/ISSUE_TEMPLATE/ticket.md (hunk 1 above)
Comment (MathewMacDougall) [reply .github/ISSUE_TEMPLATE/ticket.md:31]: I kinda like the multiple examples. There's generally at least 2 items with each ticket (do the thing and test it) and IMO it's easier to remove them than add them.

**Review by MathewMacDougall — CHANGES_REQUESTED** [review]
Should we also add a section for estimated work-hours / time to complete? Or would we rather manage that with tags?

Pros of in ticket:
* It's in the ticket, tightly coupled?
Cons:
* ???

Pros of tags:
* Can sort and search for issues by time.
Cons of tags:
* Lots of tags. Would need to group things somehow. Maybe 1 hour, 3 hour, 5 hour 10+ hour?

**Comment by garethellis0** [comment]
I think tags would be better. We don't want to be too granular with "hours" anyways, I think we'd need 5 tags at most.

File: .github/ISSUE_TEMPLATE/ticket.md (hunk 1 above)
Comment (garethellis0) [reply .github/ISSUE_TEMPLATE/ticket.md:31]: Ok, just gonna leave it as-is for now then.

**Review by MathewMacDougall — APPROVED** [review]

**Review by jonathanlew — APPROVED** [review]

**Pull request merged**